packages/plasma-new-hope/src/examples/_helpers.tsx

```diff
@@ -32,21 +32,28 @@ export function argTypesFromConfig<
     const filteredVariations = Object.entries(variations).filter(([key]) => !exclude.includes(key));
 
     const argTypes: ArgTypes = {};
+
     for (const [key, val] of filteredVariations) {
         const control: ArgTypes[''] = {
             control: {},
         };
+
         if (val.true) {
             control.control.type = 'boolean';
-            control.defaultValue = false;
+            control.table = { defaultValue: { summary: false } };
```

Comment:
[Note]:

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#no-longer-inferring-default-values-of-args